changelogs/current.yaml

```diff
@@ -37,6 +37,10 @@ new_features:
     Change HTTP status to 200 to respect the gRPC protocol. This may cause problems for incorrect gRPC clients expecting the filter
     to preserve HTTP 1.1 responses.  This behavioral change can be temporarily reverted by setting runtime guard
     ``envoy.reloadable_features.grpc_http1_reverse_bridge_change_http_status`` to false.
+- area: quic
+  change: |
+    Added QUIC protocol option :ref:`send_disable_active_migration <envoy_v3_api_field_config.listener.v3.QuicProtocolOptions.send_disable_active_migration>`
+    to discourage client endpoints from active migration.
```

Comment:
nit: To clarify, this option affects evnoy acting a server and causes it to send `disable_active_migration` which will deter client connecting to Envoy, right? Might be worth a bit of wordsmithing here.

Reply:
I took a stab at making it better; let me know what you think